server/index.js

```diff
@@ -54,6 +55,34 @@ passport.use(new FacebookStrategy({
 		});
 	}
 ));
+passport.use(new GoogleStrategy({
+	clientID: "351252051601-ta8s8cr8st2l3ubien8ldbu948g09l6j.apps.googleusercontent.com",
```

Comment:
Add these to the config files and use them afterwards. As we might need different ids for testing purpose in future so changing config file will be better alternative.